### DISCUSSION TITLE
Display additional content on post index page

### DISCUSSION THREAD

**Discussion by mayanayza** [GetPublii/Publii#1732] [Help] [answered]
I would like to add some content above the posts on the post index page, which currently is set using the /portfolio url slug in URL settings.

Essentially -

<title>
<content>
<posts index>

Is there a way to do this through the Publii app or do I need to edit something in the theme files? I tried using a page which I set to homepage with the desired content, but when I do this it no longer displays the posts index on the page.

**Comment by bjazmoore** [comment]
That is theme based.  Check out how [theme over rides work in the documentation](https://getpublii.com/dev/theme-overrides/).  If I am not mistaken the posts page is managed by the index.hbs file in the theme root - even when the post index is redirected.  Edit that file to add the content you need.

**Comment by candidexmedia** [comment]
Which theme are you using?

**Comment by mayanayza** [comment, marked as answer]
I'm using Mercury. I realized that there's an option in theme settings to add header content to the homepage which worked for me!
